Packs/ThreatConnect/Integrations/ThreatConnectV3/ThreatConnectV3.py

```diff
@@ -795,7 +799,15 @@ def tc_get_indicators_by_tag_command(client: Client, args: dict) -> None:  # pra
 
 def tc_get_indicator_command(client: Client, args: dict) -> None:  # pragma: no cover
     indicator = args.get('indicator')
-    response = tc_get_indicators_command(client, args, return_raw=True, indicator_id=indicator)  # type: ignore
+    indicator_id = ''
+    summary = ''
+    try:
+        int(indicator)  # type: ignore
+        indicator_id = indicator  # type: ignore
+    except ValueError:
+        summary = indicator  # type: ignore
```

Comment:
what's going on here? this code is not clear

Reply:
There can't be an indicator summary that is just an int so if it's an int it must be an id else it will be a summary, I do this code to support both id and summary without making breaking changes, I've added comments to the code